src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -74,6 +74,12 @@ public static IEnumerable<Example> Examples
         [Option('o', "out", Required = false, HelpText = "OutputDirectory_HelpText", ResourceType = typeof(Resources))]
         public string OutputDir { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to allow unsecure downloads.
+        /// </summary>
+        [Option("allow-unsecure-downloads", Required = false, HelpText = "AllowUnsecureDownloads_HelpText", ResourceType = typeof(Resources))]
+        public bool AllowUnsecureDownloads { get; set; }

```

Comment:
We should update the arguments section of [doc/new.md](https://github.com/microsoft/winget-create/blob/main/doc/new.md) & [doc/update.md](https://github.com/microsoft/winget-create/blob/main/doc/update.md) as well